scripts/aa-university.js

```diff
@@ -0,0 +1,18 @@
+export default function registerAAUniversity() {
+  const firstName = document.querySelector('input[name="FirstName"]');
+  const lastName = document.querySelector('input[name="LastName"]');
+  const email = document.querySelector('input[name="Email"]');
+  const country = document.querySelector('select[name="Country"]');
+
+  fetch('https://us-central1-adobe---aa-university.cloudfunctions.net/register', {
+    method: 'POST',
+    body: JSON.stringify({
+      first_name: firstName?.value,
+      last_name: lastName?.value,
+      email: email?.value,
+      university: 'none',
+      country: country?.value,
+    }),
+  })
+    .catch((error) => window.lana.log(`Marketo AA University Error: ${error}`));
```

Comment:
Any chance `window.lana` might not exist at this point, justifying the use of some optional chaining?

Reply:
That's a good thought, but I don't think so. This is being run after loadArea and we don't even use optional chaining there within block code